Gemfile

```diff
@@ -62,7 +62,7 @@ gem "puma", "~> 3.12"
 
 # Rails request timeout, needed if running on Heroku-
 # https://devcenter.heroku.com/articles/request-timeout
-gem "rack-timeout"
+gem "rack-timeout", require: "rack/timeout/base"
```

Comment:
@Swastikyadav This change has nothing to do with `eslint`. Please remove it from this PR and put in a different PR.

Reply:
I have removed the unnecessary change in the gemfile. We can merge this PR now.